File: 09/CS/CustomControls/09CustomControls/Client/UserCode/WebBrowserCtrlDetail910.cs
using System;
using System.Linq;
using System.IO;
using System.IO.IsolatedStorage;
using System.Collections.Generic;
using Microsoft.LightSwitch;
using Microsoft.LightSwitch.Framework.Client;
using Microsoft.LightSwitch.Presentation;
using Microsoft.LightSwitch.Presentation.Extensions;
using CentralControlsCS;

namespace LightSwitchApplication
{
    public partial class WebBrowserCtrlDetail910
    {
        partial void Hyperlink_Loaded(bool succeeded)
        {
            // Write your code here.
            this.SetDisplayNameFromEntity(this.Hyperlink);
        }

        partial void Hyperlink_Changed()
        {
            // Write your code here.
            this.SetDisplayNameFromEntity(this.Hyperlink);
        }

        partial void WebBrowserCtrlDetail910_Saved()
        {
            // Write your code here.
            this.SetDisplayNameFromEntity(this.Hyperlink);
        }

        partial void WebBrowserCtrlDetail910_Activated()
        {
            // Write your code here.
            Property1 = "Type in a URL and the control will update iteself automatically";
            IContentItemProxy ctrl = this.FindControl("URL");
            ctrl.SetBinding(CentralControlsCS.WebBrowserControl.URIProperty, "Value", System.Windows.Data.BindingMode.OneWay );

        }
    }
}

File: 09/VB/09CustomControls/Client/UserCode/WebBrowserCtrlDetail910.vb

Namespace LightSwitchApplication

    Public Class WebBrowserCtrlDetail910

        Private Sub Hyperlink_Loaded(succeeded As Boolean)
            ' Write your code here.
            Me.SetDisplayNameFromEntity(Me.Hyperlink)
        End Sub

        Private Sub Hyperlink_Changed()
            ' Write your code here.
            Me.SetDisplayNameFromEntity(Me.Hyperlink)
        End Sub

        Private Sub WebBrowserCtrlDetail910_Saved()
            ' Write your code here.
            Me.SetDisplayNameFromEntity(Me.Hyperlink)
        End Sub

        Private Sub WebBrowserCtrlDetail910_Activated()
            ' Write your code here.

            Property1 = "Type in a URL and the control will update iteself automatically"
            Dim ctr As IContentItemProxy = Me.FindControl("URL1")
            ctr.SetBinding(CentralControls.WebBrowserControl.URIProperty, "Value", Windows.Data.BindingMode.OneWay)

        End Sub
    End Class

End Namespace